dogstatsd.py

```diff
@@ -62,6 +62,10 @@
 DOGSTATSD_FLUSH_INTERVAL = 10
 DOGSTATSD_AGGREGATOR_BUCKET_SIZE = 10
 
+# Dogstatsd UDP packet size in K
+UDP_PACKET_SIZE_OPTIONS = [8, 16, 32, 64]
+DEFAULT_UDP_PACKET_SIZE = 8
```

Comment:
can we change the names of these to `DEFAULT_MAX_UDP_PACKET_SIZE_KB`, currently it's a bit misleading.